Reports_Queries/Jasper/Annex_17/sub_report_coordinates.sql

```diff
@@ -1,29 +1,33 @@
-WITH 
+WITH
 parametros AS (
   SELECT
-    1522 	AS poligono_t_id, --$P{id}
-     4 		AS criterio_punto_inicial, --tipo de criterio para seleccionar el punto inicial del terreno, valores posibles: 1,2 parametrizar $P{criterio_punto_inicial} 
-    4		AS criterio_observador, --1: Centroide, 2: Centro del extent, 3: punto en la superficie, 4: Punto mas cercano al centroide dentro del poligono
-    true	AS incluir_tipo_derecho --Mostrar el tipo de derecho de cada interesado (booleano)
+    1432 	AS terreno_t_id, --$P{id}
+     2 		AS criterio_punto_inicial, --tipo de criterio para seleccionar el punto inicial del terreno, valores posibles: 1,2 parametrizar $P{criterio_punto_inicial}
+     4		AS criterio_observador --1: Centroide, 2: Centro del extent, 3: punto en la superficie, 4: Punto mas cercano al centroide dentro del poligono
 ),
-t AS ( --Orienta los vertices del terreno en sentido horario
-	SELECT t_id, ST_ForceRHR(geometria) as geometria FROM ladm_lev_cat_v1.lc_terreno AS t, parametros WHERE t.t_id = poligono_t_id --parametrizar $P!{datasetName}
+t AS (
```

Comment:
Next time, please do not remove this kind of documentation. It's useful.

Reply:
There, I'll be more careful. Thanks